src/protocols/MeterSML.cpp

```diff
@@ -97,6 +97,12 @@ MeterSML::MeterSML(std::list<Option> options)
 		/* using default value if not specified */
 		_use_local_time = false;
 	}
+	try {
+		_direction_from_status = optlist.lookup_bool(options, "direction_from_status");
```

Comment:
is the name of this flag sensible?
it should maybe include "power" too, as we apply it to the power channel.
also it could include "sign"....

"derivce_power_sign_from_direction_bit_in_status_field" would be correct, but is a bit excessive.